spotify.txt

```diff
@@ -0,0 +1,13 @@
+﻿[Section]
+Name = Spotify
+Version = 1.0.29.92
+License = Freeware
+Description = Spotify is one of the most popular services for streaming music and includes both a free and paid plan. The Spotify client has dropped support for XP, but this archived version still supports it.
+Size = 47.7 MB MiB
+Category = 1
+URLSite = https://www.spotify.com/us/
+URLDownload = https://bennottelling.com/ros/apps/SpotifyFullSetup1.0.29.92.exe
```

Comment:
https://filehippo.com/fr/download_spotify/68194/
less likely to go down

Reply:
That's true, but it's not a direct link to the file, plus I've got cloudflare on my server so that should help with uptime.